Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Tim Kittel", "Jan Liße", "Sabine Auer"]
 version = "2.4.1"
 
 [deps]
+Crayons = "a8cc5b0e-0ffa-5ad4-8c14-923d3ee1735f"
```

Comment:
It seems you locally added this dep. Probably it can be removved?

Reply:
Yeah I have added locally because it was needed in the runtests.jl! I can remove it